### DISCUSSION TITLE
Remember line of file

### DISCUSSION THREAD

**Discussion by milliams** [helix-editor/helix#4199] [Ideas] [answered]
Before opening an issue for this enhancement, I wanted to put it here first.

In VIM, I can open a file and navigate to a certain line in it and then close VIM. When I restart VIM with that file, it will automatically navigate me back to the same line.

In Helix it will always reopen on the first line of the file, requiring me to navigate back to where I was working.

I think it would be useful if Helix had the same behaviour as VIM here and remembered the open line.

Thoughts?

**Comment by dead10ck** [comment, marked as answer]
#401

**Reply by milliams** [reply]
Thanks, I tried searching of course but clearly didn't use the right combination of terms.
